qovery/data_source_helm.go

```diff
@@ -274,6 +280,196 @@ func (d helmDataSource) Schema(_ context.Context, _ datasource.SchemaRequest, re
 					},
 				},
 			},
+			"timeout_sec": schema.Int64Attribute{
+				Description: "Helm timeout in second",
+				Optional:    true,
+				Computed:    true,
+			},
+			"auto_preview": schema.BoolAttribute{
+				Description: "Specify if the environment preview option is activated or not for this helm.",
+				Optional:    true,
+				Computed:    true,
+			},
+			"auto_deploy": schema.BoolAttribute{
+				Description: " Specify if the service will be automatically updated on every new commit on the branch.",
```

Comment:
_suggestion_: remove extra starting space + extra `the` => `Specify if service will be automatically updated on every new commit on branch.`

Reply:
done